lambda-function/src/utils/measureConversionUtils.ts

```diff
@@ -214,13 +214,20 @@ const getMeasuretypes = (measuretypes: Array<MeasureType>): Array<MeasureGroupTy
         types.push(MeasureGroupTypes.STRUCTURE);
         break;
       default:
-        types.push(MeasureGroupTypes.OUTCOME);
+        if (!alreadyHasDefaultMeasureTye(types)) {
```

Comment:
because eventually, it is just an array of strings, probably `array.includes` or `array.indexOf` would work

Reply:
Ok thanks changed to use includes.